app/src/main/res/layout/item_search_result.xml

```diff
@@ -61,12 +61,38 @@
         app:layout_constraintTop_toBottomOf="@+id/page_list_item_redirect"
         tools:text="Lorem ipsum" />
 
+    <ImageView
```

Comment:
Looks like the view and the `TabCountsView` are not aligning to the center correctly. You can see the issue when other results have thumbnails and these views are slightly aligning to the left.

Reply:
![n](https://user-images.githubusercontent.com/9540770/92058907-a3ce8080-ed44-11ea-99a2-290fa2c446d8.png)
@cooltey Do you mean in cases like. this. history icon and TabCountsView arent aligned?

Reply:
> @cooltey Do you mean in cases like this history icon and TabCountsView arent aligned?


You'll see it when there are no thumbnails on the histroy/tab items.
![Screenshot (Sep 3, 2020 08 32 47)](https://user-images.githubusercontent.com/2435576/92136105-7b846780-edc0-11ea-9652-3f8e8d88280a.png)